1_html-css/index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Andrii Yaroshenko CV</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+  </head>
+  <body>
+    <section class="container">
+      <header>
+      <nav class="box">
+        <div class="menu">
+          <a href="#about">About</a>
+          <a href="#portfolio">Portfolio</a>
+          <a href="#experience">Experience</a>
+          <a href="#contacts">Contacts</a>
+        </div>
+      </nav>
+      <div class="info">
+        <img class="avatar" src="/ay.jpg" alt="Andrii Yaroshenko" />
+        <div class="info-text">
+          <h1>Hey 👋</h1>
+          <div class="wrapper">
+            <p class="text typewriter-animation">
+              I'm Andrii Yaroshenko 
+            </p>
+          </div>
+          <p>
+            I am a beginner web developer and student at the HackYourFuture in
+            Copenhagen. I am excited to learn web development.
+          </p>
+        </div>
+      </div></header>
+      <div class="about" id="about">
+        <h2>About Me</h2>
+        <p>While I might be new to the world of web development, my enthusiasm and eagerness to learn are boundless.</p>
+        <p>I'm currently exploring the fascinating realm of HTML, CSS, and JavaScript, and every line of code I write feels like a step towards unlocking a new realm of creativity. I'm embracing the challenges that come with this adventure and am dedicated to building a strong foundation in web development.</p>
+        <p>Join me as I document my learning process, share my small wins, and maybe a few frustrations too. This website is a testament to my commitment to growth and the exciting path ahead.</p>
+        <p>Feel free to <a href="mailto:ayaroshenkohyf@gmail.com">connect with me!</a></p>
+      </div>
+      <div class="portfolio" id="portfolio">
+        <h2>Portfolio</h2>
```

Comment:
Remember to include the survey form as the first portfolio project :  https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form